runelite-client/src/main/java/net/runelite/client/plugins/banktags/TagManager.java

```diff
@@ -169,6 +178,35 @@ public void removeTag(int itemId, String tag)
 		}
 	}
 
+	public boolean itemWithinPriceRange(Item item)
```

Comment:
Nothing is done with tag search, so why is this here?

Reply:
I wasn't sure where to put it.
I considered the price of the item as a sort of "tag".